tests/unit/test_collect_schemas.py

```diff
@@ -97,7 +97,7 @@ class ProviderSchema(DataBagSchema):
     (
         (dedent("""Foo2: int=1"""), "Foo2"),
         (dedent("""Bar: str='baz'"""), "Bar"),
-        (dedent("""Baz: list[int]=[1,2,3]"""), "Baz"),
+        (dedent("""Baz: list=[1,2,3]"""), "Baz"),
```

Comment:
```suggestion
        (dedent("""Baz: List[int]=[1,2,3]"""), "Baz"),
```
maybe this was the issue?